examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
Can we add a landing page (similar to https://beam.apache.org/get-started/try-apache-beam/) which jumps to this collab page?

Reply:
That would be great! Is this something @robertwb can arrange?

Reply:
I could be wrong, but I imagine it's just a matter of making a new page similar to that one and adding to the TOC

Reply:
Tech Writers can probably help answer this question too

Reply:
Me or @svetakvsundhar will work on this once this PR is merged.

Reply:
Filed https://github.com/apache/beam/issues/27450

Reply:
SGTM, can you use the new YAML tags that Jeff created? https://bugdashboard.corp.google.com/app/tree;dashboardId=551658